### ISSUE TITLE
Eigene paper-size ist problematisch

### ISSUE THREAD

**Issue by The-Compiler** [ost-fh/Latex-Beamer-Theme#6]
Das LaTeX `beamer` package funktioniert so, dass es eine Paper-Size von 16cm x 9cm setzt (sofern `aspectratio=169` gegeben ist). Dies ist so gemacht, damit alle anderen LaTeX-Packages trotzdem Output liefern können, der auf ein normales Papier passt, und er wird halt entsprechend passend auf einen Beamer-Slide hochskaliert. So machen dann auch die LaTeX-Schriftgrössen weiterhin sinn, weil das Ergebnis trotzdem noch sinnvoll lesbar ist.

Das Theme setzt die Papiergrösse auf 33.88cm auf 19.05cm. Zum einen entspricht das nicht 16:9, aber die Slides wären dann z.B. 1920x1079.57px, sollte also in der Praxis wohl nicht drauf an kommen. Das grössere Problem ist, dass sich andere Packages und Schriftgrössen kaum mehr verwenden lassen:

![image](https://user-images.githubusercontent.com/625793/93023923-c721e700-f5f2-11ea-8e64-385dc01d5477.png)

![image](https://user-images.githubusercontent.com/625793/93023945-dbfe7a80-f5f2-11ea-916b-8de6aec96821.png)

Gibt es einen Grund für diese (für das beamer-Package) sehr untypische Wahl? Ging es einfach darum, dass ihr die Abstände aus dem Powerpoint-Design übernehmen konntet? IMHO würde es viel mehr Sinn machen, das Theme entsprechend anzupassen dass es wie andere Beamer-Themes funktioniert anstatt zu versuchen, alle anderen Packages/Schriftgrössen/... entsprechend hinzubiegen.

**Comment by misto** [comment]
Ja, das war der Grund, da es viele Elemente zu positionieren gibt, und Powerpoint da die Referenz war (auch die Papersize). Können wir aber von mir aus auch gerne anpassen.
